usr/share/rear/skel/default/etc/scripts/system-setup.d/58-start-dhclient.sh

```diff
@@ -1,11 +1,12 @@
 # start dhclient daemon script
 #
-# check if we have USE_DHCLIENT=y, if not then we run 60/62 scripts
-[[ -z "$USE_DHCLIENT"  ]] && return
+# Skip execution if USE_DHCLIENT is empty or set to 'false'.
+[[ -z "$USE_DHCLIENT" ]] || is_false $USE_DHCLIENT && return
 
 # with USE_STATIC_NETWORKING no networking setup via DHCP must happen
 # see default.conf: USE_STATIC_NETWORKING overrules USE_DHCLIENT
-test "$USE_STATIC_NETWORKING" && return
+# Skip execution if USE_STATIC_NETWORKING is set to any string except 'false'.
+[[ -z "$USE_STATIC_NETWORKING" ]] || is_false $USE_STATIC_NETWORKING || return
```

Comment:
I think
```
! is_true $USE_STATIC_NETWORKING && return
```
makes the code simpler as in
https://github.com/rear/rear/pull/2325/files#r374719525